tools/wheel_resolver/wheel.py

```diff
@@ -72,18 +68,3 @@ def url(
 
 def _is_wheel(url: str) -> bool:
     return url.endswith(".whl")
-
-
-def _is_compatible(url: str, tags: typing.List[str]) -> bool:
```

Comment:
We can remove `_is_compatible` because `distlib` already checks for tag compatibility